src/pages/portfolio/AccountOverviewSection.tsx

```diff
@@ -128,11 +139,16 @@ export const AccountOverviewSection = () => {
                   section.amount != null &&
                   totalValue != null &&
                   section.amount > 0 && (
-                    <$LineSegment
+                    <WithTooltip
                       key={section.id}
-                      tw="h-full"
-                      $color={section.color}
-                      $widthPercent={section.amount / totalValue}
+                      tooltipStringTitle={`${section.label}: ${formatNumberOutput(section.amount, OutputType.Fiat, { decimalSeparator, groupSeparator, selectedLocale })} (${formatNumberOutput(section.amount / totalValue, OutputType.Percent, { fractionDigits: 0, decimalSeparator, groupSeparator, selectedLocale })})`}
```

Comment:
bruh

Reply:
okay okay I can make it not a billion character long line